starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  font-family: ‘Source Sans Pro’, sans-serif;
+  font-weight: 100;
+  font-size: 14px;
+  color: #333;
+  width: 100%;
+  height: 100%;
+}
+
```

Comment:
Nice design, looks good, I would have added padding to the body so it looks a bit more like in the image. Try adding something like this in the body ```padding: 0 120px```